compiler/modules/nback/number.bal

```diff
@@ -20,6 +21,24 @@ final RuntimeFunction decimalNegFunction = {
     attrs: ["readonly"]
 };
 
+final RuntimeFunction decimalToIntFunction = {
+    name: "decimal_to_int",
+    ty: {
+        returnType: llvm:structType(["i64", "i1"]),
```

Comment:
This struct is used here and in floatToInt, so construct it once and stick it in a final module-level variable.

Reply:
Changed